### ISSUE TITLE
Ili9xxx support

### ISSUE THREAD

**Issue by styphonthal** [landonr/lilygo-tdisplays3-esphome#65]
Does the tdisplays3 non amoled version have ili9xxx support? From what I can tell no?

**Comment by guillempages** [comment]
No, the non-amoled t-display-s3 does not use "standard" ili9xxx, so the standard esphome ili9xxx component cannot be used to control this display.
